types/globalTask.ts

```
@@ -0,0 +1,14 @@
+import { Prisma } from '@prisma/client'
+
+// Raw task type from Prisma
+export type PrismaTask = Prisma.GlobalTaskGetPayload<{}>
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)